openapi.yaml

```diff
@@ -89,6 +89,23 @@ paths:
         default:
           $ref: '#/responses/InternalServerError'
 
+  /api/v1/log/timestampCertChain:
```

Comment:
should this be `/api/v1/tsr/certchain` maybe?

Reply:
I would rather see this end up as `/api/v1/timestamp/certchain` or perhaps update this and the public key endpoint to be grouped together?